src/main/drivers/serial_softserial.c

```diff
@@ -118,9 +118,13 @@ static void serialEnableCC(softSerial_t *softSerial)
 {
 #ifdef USE_HAL_DRIVER
     TIM_CCxChannelCmd(softSerial->timerHardware->tim, softSerial->timerHardware->channel, TIM_CCx_ENABLE);
+#else
+#ifdef AT32F435
```

Comment:
As we are removing CPU specific macro's guess we need to port this file to AT32 but still see no need for softserial now we have 6 UARTs.